kolibri/plugins/user/assets/src/views/profile-page/index.vue

```diff
@@ -102,7 +102,7 @@
     },
     methods: {
       hasPrivilege(privilege) {
-        return this.privileges[privilege];
+        return this.facilityConfig[privilege];
```

Comment:
I know you didn't write this (I did. Sorry.), but might be best to write a separate computed prop for each permission now.